spring-boot-autoconfigure/pom.xml

```diff
@@ -497,5 +497,11 @@
 			<artifactId>slf4j-jdk14</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.cxf</groupId>
+			<artifactId>cxf-bundle</artifactId>
+			<version>3.0.0-milestone2</version>
```

Comment:
Rather than specifying a version here, dependency management should be added to `spring-boot-dependencies`. Also, what made you choose 3.0.0-milestone2. Unless we're certain of the date for 3.0.0 reaching GA, I'd prefer to go for an earlier version.